src/chains/ethereum/options/src/miner-options.ts

```diff
@@ -158,6 +158,18 @@ export type MinerConfig = {
       type: Data;
       hasDefault: true;
     };
+
+    /**
+     * Minimum price bump percentage needed to replace a transaction that already exists in the transaction pool.
+     *
+     * @defaultValue ""
+     */
+    priceBump: {
+      type: bigint;
+      rawType: number | bigint;
```

Comment:
Since the `cliType` is string and the type is `bigint` (normalized by calling `BigInt(rawType)`) we will accept things like `0o10` (8n) and `0n1111` (15), which I don't think we accept anywhere else.

Additionally, the `rawType` should probably include `string` (since `cliType` is string), so that we can accept the same hex strings on the command line and in typescript (`ganache --miner.priceBump 0xff` and ganache.provider({miner:{priceBump: "0xff"}}))`.

Reply:
I added `string` to `rawType`. I don't think any changes need to be made to the normalization, this will accept  `0xf`, `0b11`, and `20` values, as do other options (i.e. `--miner.defaultGasPrice`)